### PR TITLE
feat: add MongoDB container for local development

### PR DESCRIPTION
**Pull request by wagnerjfr**
Updated README.md to provide the MongoDB DB_URL for local setup.
Modified docker-compose.dev.yml to configure MongoDB service for local development.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,10 @@ cd nyrkio
 git submodule init
 git submodule update
 
-cat <<<END  > .env.backend
+cat > .env.backend << END
+## When using docker-compose.dev.yml with MongoDB running as a container.
+#DB_URL=mongodb://mongodb:27017/mongodb
+
 #DB_URL=mongodb://localhost:27017/nyrkiodb
```
Comment (mfleming) [inline README.md:37]: I think we should just make running mongo in a container the default and remove all other `DB_URL` examples.

File: docker-compose.dev.yml [hunk 2]
```diff
@@ -18,4 +26,4 @@ services:
       - "80:80"
     depends_on:
       - backend
-    restart: always
+    restart: always
```
Comment (mfleming) [inline docker-compose.dev.yml:29]: Unintentional change? Looks like you removed the newline at the end of the file.

**Review by mfleming — CHANGES_REQUESTED** [review]
Overall looks good. Just a couple of minor changes requested.

Thanks for your contribution!

**Review by wagnerjfr — COMMENTED** [review]
Addressed the suggestions

**Comment by wagnerjfr** [comment]
Let me know if the change is ok

**Review by mfleming — APPROVED** [review]
lgtm thanks!

**Pull request merged**